### PR TITLE
Enhance "About GitHub Advanced Security" page

### PR DESCRIPTION
**Pull request by lucie-docs**
### Why:

Closes https://github.com/github/docs-content/issues/6200 

### What's being changed:

Added a table breaking down GHAS availabilty for public and private repos for ftp and ghec.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by lucie-docs** [comment]
Ok, this PR is good to go now!

**Comment by ramyaparimi** [comment]
@lucie-docs Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 1]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
 - **Security overview** - Review the security configuration and alerts for an organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
 {% endif %}
 
+{% ifversion fpt %}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:39]: This sentence can be used for both FPT and GHEC so suggesting the following (also used the variable for GitHub Advanced Security):
```suggestion
{% ifversion fpt or ghec%}
The table below summarizes the availability of {% data variables.product.prodname_GH_advanced_security %} features for public and private repositories.

{% elsif fpt %}
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 2]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+The table below summarizes the availability of GitHub Advanced Security features on public and private repositories.
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:40]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 3]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+
+|                   | Public repository           | Private repository without Advanced Security | Private repository with Advanced Security |
+| :-----------------: | :---------------------------: | :--------------------------------------------: | :-----------------------------------------: |
+| Code scanning     | Yes                         | No                                           | Yes                                        |
+| Secret scanning   | Yes **(limited functionality only)** | No                                           | Yes                                       |
+| Dependency review | Yes                         | No                                           | Yes                                       |
+{% endif %}
+
+{% ifversion ghec %}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:49]:
```suggestion
{% elsif ghec %}
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 4]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% ifversion fpt %}
+The table below summarizes the availability of GitHub Advanced Security features on public and private repositories.
+
+|                   | Public repository           | Private repository without Advanced Security | Private repository with Advanced Security |
+| :-----------------: | :---------------------------: | :--------------------------------------------: | :-----------------------------------------: |
+| Code scanning     | Yes                         | No                                           | Yes                                        |
+| Secret scanning   | Yes **(limited functionality only)** | No                                           | Yes                                       |
+| Dependency review | Yes                         | No                                           | Yes                                       |
+{% endif %}
+
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:48]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 5]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% ifversion ghec %}
+The table below summarizes the availability of GitHub Advanced Security features on public and private repositories.
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:50]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 6]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% data variables.product.prodname_GH_advanced_security %} features are enabled for all public repositories on {% data variables.product.prodname_dotcom_the_website %}. Organizations that use {% data variables.product.prodname_ghe_cloud %} with {% data variables.product.prodname_advanced_security %} can additionally enable these features for private and internal repositories. They also have access to an organization-level security overview. For more information, see the [{% data variables.product.prodname_ghe_cloud %} documentation](/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security#enabling-advanced-security-features).
+{% endif %}
+
+{% ifversion ghec %}
+{% data variables.product.prodname_GH_advanced_security %} features are enabled for all public repositories on {% data variables.product.prodname_dotcom_the_website %}, except for the security overview. Organizations that use {% data variables.product.prodname_ghe_cloud %} with {% data variables.product.prodname_advanced_security %} can additionally enable these features for private and internal repositories. They also have access to an organization-level security overview.
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:67]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 7]
```diff
@@ -36,10 +36,35 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
 {% endif %}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:68]:
```suggestion
```

**Review by mchammer01 — DISMISSED** [review]
@lucie-docs - thanks for adding these tables ✨ 
I've made some simplifications to the versioning and I'm going to apply my suggestions shortly.

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 8]
```diff
@@ -36,12 +36,33 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
 - **Security overview** - Review the security configuration and alerts for an organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
 {% endif %}
 
+{% ifversion fpt or ghec%}
+The table below summarizes the availability of {% data variables.product.prodname_GH_advanced_security %} features for public and private repositories.
+
+{% elsif fpt %}
+
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:43]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 9]
```diff
@@ -36,12 +36,33 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+|                   | Public repository           | Private repository without {% data variables.product.prodname_advanced_security %} | Private repository with {% data variables.product.prodname_advanced_security %} |
+| :-----------------: | :---------------------------: | :--------------------------------------------: | :-----------------------------------------: |
+| Code scanning     | Yes                         | No                                           | Yes                                        |
+| Secret scanning   | Yes **(limited functionality only)** | No                                           | Yes                                       |
+| Dependency review | Yes                         | No                                           | Yes                                       |
+{% elsif ghec %}
+
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:50]:
```suggestion
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 10]
```diff
@@ -36,12 +36,33 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
 - **Security overview** - Review the security configuration and alerts for an organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
 {% endif %}
 
+{% ifversion fpt or ghec%}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:39]:
```suggestion
{% ifversion fpt or ghec %}
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 11]
```diff
@@ -36,12 +36,33 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% data variables.product.prodname_GH_advanced_security %} features are enabled for all public repositories on {% data variables.product.prodname_dotcom_the_website %}{% ifversion ghec %}, except for the security overview{% endif %}. Organizations that use {% data variables.product.prodname_ghe_cloud %} with {% data variables.product.prodname_advanced_security %} can additionally enable these features for private and internal repositories. They also have access to an organization-level security overview. For more information, see the [{% data variables.product.prodname_ghe_cloud %} documentation](/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security#enabling-advanced-security-features).
 {% endif %}
 
+
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:65]:
```suggestion
```

**Review by mchammer01 — COMMENTED** [review]
More tidying up

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 12]
```diff
@@ -36,12 +36,33 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% data variables.product.prodname_GH_advanced_security %} features are enabled for all public repositories on {% data variables.product.prodname_dotcom_the_website %}{% ifversion ghec %}, except for the security overview{% endif %}. Organizations that use {% data variables.product.prodname_ghe_cloud %} with {% data variables.product.prodname_advanced_security %} can additionally enable these features for private and internal repositories. They also have access to an organization-level security overview. For more information, see the [{% data variables.product.prodname_ghe_cloud %} documentation](/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security#enabling-advanced-security-features).
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:60]: Reinstate initial versioning
```suggestion
{% data variables.product.prodname_GH_advanced_security %} features are enabled for all public repositories on {% data variables.product.prodname_dotcom_the_website %}{% ifversion ghec %}, except for the security overview{% endif %}. Organizations that use {% data variables.product.prodname_ghe_cloud %} with {% data variables.product.prodname_advanced_security %} can additionally enable these features for private and internal repositories. They also have access to an organization-level security overview. {% ifversion fpt %}For more information, see the [{% data variables.product.prodname_ghe_cloud %} documentation](/enterprise-cloud@latest/get-started/learning-about-github/about-github-advanced-security#enabling-advanced-security-features).{% endif %}
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 13]
```diff
@@ -36,10 +36,28 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+{% ifversion fpt or ghec %}
+The table below summarizes the availability of {% data variables.product.prodname_GH_advanced_security %} features for public and private repositories.
+
+{% elsif fpt %}
+|                   | Public repository           | Private repository without {% data variables.product.prodname_advanced_security %} | Private repository with {% data variables.product.prodname_advanced_security %} |
+| :-----------------: | :---------------------------: | :--------------------------------------------: | :-----------------------------------------: |
+| Code scanning     | Yes                         | No                                           | Yes                                        |
+| Secret scanning   | Yes **(limited functionality only)** | No                                           | Yes                                       |
+| Dependency review | Yes                         | No                                           | Yes                                       |
+{% elsif ghec %}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:48]:
```suggestion
{% endif %}
{% ifversion ghec %}
```

File: content/get-started/learning-about-github/about-github-advanced-security.md [hunk 14]
```diff
@@ -36,10 +36,28 @@ A {% data variables.product.prodname_GH_advanced_security %} license provides th
...
+|                   | Public repository           | Private repository without {% data variables.product.prodname_advanced_security %} | Private repository with {% data variables.product.prodname_advanced_security %} |
+| :-----------------: | :---------------------------: | :--------------------------------------------: | :-----------------------------------------: |
+| Code scanning     | Yes                         | No                                           | Yes                                        |
+| Secret scanning   | Yes **(limited functionality only)** | No                                           | Yes                                       |
+| Dependency review | Yes                         | No                                           | Yes                                       |
+| Security overview | No                          | No                                           | Yes                                          |
+{% endif %}
```
Comment (mchammer01) [inline content/get-started/learning-about-github/about-github-advanced-security.md:55]:
```suggestion
{% endif %}

{% endif %}
```

**Review by mchammer01 — COMMENTED** [review]
Hopefully fixing versioning

**Review by mchammer01 — APPROVED** [review]
LGTM ✨ 
Versioning all working as expected, wohoo!

**Comment by Satkhiramch** [comment]
> 

PR

**Comment by Satkhiramch** [comment]
a5c3785

**Pull request merged**